script.js

```diff
@@ -0,0 +1,13 @@
+// var userCountry = $("country-choice");
+var queryURL = "https://api.covid19api.com/live/country/italy/status/confirmed/date/2020-03-21T13:13:30Z";
+
+console.log(queryURL)
+
+$.ajax ({
+    url: queryURL,
+    method: "GET"
+}) .then(function(response){
+    console.log(response);
+    var currentCases = response[response.length - 1].Active;
```

Comment:
The variable here should be var newCase (integer) and depicts number of new cases that day
If that is not available, lets talk about it later.